stylesheets/github-light.css

```diff
@@ -5,7 +5,7 @@
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at
 
-       http://www.apache.org/licenses/LICENSE-2.0
+       https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please revert this.

This is a license text from a 3rd party component, which we cannot change.